### PR TITLE
FLAG-43: Migrate SQLDIFF to liquibase changesets

### PR DESCRIPTION
**Pull request by slubwama**
This PR Changes the way how patient flags deploys tables into the database from SQLDIFF to liquibase changesets
Ticket Number[ FLAG-43](https://issues.openmrs.org/browse/FLAG-43)

### CODE REVIEW

**Comment by slubwama** [comment]
@dkayiwa  @ssmusoke  requesting for reviews

**Comment by dkayiwa** [comment]
Can you include the ticket id in your commit message?

**Comment by slubwama** [comment]
@dkayiwa Thats is done

**Comment by dkayiwa** [comment]
@slubwama thats not true

**Comment by slubwama** [comment]
@dkayiwa now Its true

File: api/src/main/resources/liquibase.xml [hunk 1]
```diff
@@ -4,6 +4,309 @@
...
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="name" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="description" type="VARCHAR(1000)"/>
+            <column defaultValueNumeric="0" name="creator" type="INT">
+                <constraints nullable="false"/>
+            </column>
+            <column defaultValue="0000-00-00 00:00:00" name="date_created" type="datetime">
```
Comment (ssmusoke) [inline api/src/main/resources/liquibase.xml]: @slubwama Remove defaultValue for date created

File: api/src/main/resources/liquibase.xml [hunk 2]
```diff
@@ -4,6 +4,309 @@
                    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
                    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9
                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+    <changeSet author="slubwama" id="patientflags-1554735240623-1">
```
Comment (ssmusoke) [inline api/src/main/resources/liquibase.xml]: @slubwama Add a comment tag to each of the changesets describing what each of them is doing

File: api/src/main/resources/liquibase.xml [hunk 3]
```diff
@@ -4,6 +4,309 @@
...
+                <constraints nullable="false"/>
+            </column>
+            <column name="evaluator" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="description" type="VARCHAR(1000)"/>
+            <column defaultValueNumeric="0" name="creator" type="INT">
+                <constraints nullable="false"/>
+            </column>
+            <column defaultValue="0000-00-00 00:00:00" name="date_created" type="datetime">
```
Comment (ssmusoke) [inline api/src/main/resources/liquibase.xml]: @slubwama remove defaultValue

File: api/src/main/resources/liquibase.xml [hunk 4]
```diff
@@ -4,6 +4,309 @@
...
+                <constraints nullable="false"/>
+            </column>
+            <column name="rank" type="INT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="description" type="VARCHAR(1000)"/>
+            <column defaultValueNumeric="0" name="creator" type="INT">
+                <constraints nullable="false"/>
+            </column>
+            <column defaultValue="0000-00-00 00:00:00" name="date_created" type="datetime">
```
Comment (ssmusoke) [inline api/src/main/resources/liquibase.xml]: @slubwama remove defaultValue

File: api/src/main/resources/liquibase.xml (hunk 2 above)
Comment (slubwama) [reply api/src/main/resources/liquibase.xml]: @ssmusoke done

**Comment by dkayiwa** [comment]
Can you squash your commits into one?

**Comment by dkayiwa** [comment]
@slubwama in addition to Steven's comments, can you also include the ticket id in your commit message?

**Comment by ssmusoke** [comment]
@slubwama This PR contains lots of files which are not related to this PR - owa, HTML and JS

**Pull request closed without merging**